### PR TITLE
Remove TB references

### PR DESCRIPTION
**Pull request by prycrane**
Remove Treasury Board references from DTO material.

**Preview**

- https://deploy-preview-354--design-system-canada-ca.netlify.app

**French PR**

- https://github.com/canada-ca/systeme-conception/pull/234

### CODE REVIEW

**Comment by prycrane** [comment]
@delisma - I removed the underscored files.  They are now out of date and i don't want to update ROTed files.  They are still available in the history.  Is that okay?

**Review by delisma — APPROVED** [review]
Pending French PR approval

File: architecture/mandatory-elements.md [hunk 1]
```diff
@@ -5,8 +5,7 @@ dateModified: 2023-03-28
 description: "Mandatory elements for all Government of Canada webpages."
 layout: default
 title: "Mandatory elements of the Canada.ca design"
----
-<div class="mrgn-tp-md mrgn-bttm-sm brdr-bttm">
+---<div class="mrgn-tp-md mrgn-bttm-sm brdr-bttm">
```
Comment (nschonni) [inline architecture/mandatory-elements.md]: This breaks the frontmatter, the DIV needs to be on its own line. This happened in a few files

Comment (delisma) [inline architecture/mandatory-elements.md]:
```suggestion
---
<div class="mrgn-tp-md mrgn-bttm-sm brdr-bttm">
```

File: architecture/organizing-content.md [hunk 2]
```diff
@@ -4,9 +4,7 @@ date: 2017-10-05
 dateModified: 2023-03-28
 description: "How to organize your content on Canada.ca"
 title: "Organizing content on Canada.ca"
----
-<p class="gc-byline"><strong>From: <a href="https://www.canada.ca/en/treasury-board-secretariat.html">Treasury Board of Canada Secretariat</a></strong></p>
-<div class="mrgn-tp-md mrgn-bttm-sm brdr-bttm">
+---<div class="mrgn-tp-md mrgn-bttm-sm brdr-bttm">
```
Comment (delisma) [inline architecture/organizing-content.md]:
```suggestion
---
<div class="mrgn-tp-md mrgn-bttm-sm brdr-bttm">
```

**Review by delisma — CHANGES_REQUESTED** [review]

File: architecture/mandatory-elements.md (hunk 1 above)
Comment (prycrane) [reply architecture/mandatory-elements.md]: fixed

**Review by donohuech — APPROVED** [review]

**Review by delisma — APPROVED** [review]

**Pull request merged**